engine/schema/migrations/20170728040000_change_table_engine_and_encoding.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+use Phinx\Db\Adapter\MysqlAdapter;
+use Phinx\Migration\AbstractMigration;
+
+class ChangeTableEngineAndEncoding extends AbstractMigration {
```

Comment:
One other option is to rename all old tables by adding a prefix or suffix, creating new schema and then migrating all values from old tables. It might be a bit cleaner